packages/expo-dev-client/README.md

```diff
@@ -1,12 +1,13 @@
 # expo-dev-client
 
-This is a preview version of the Expo Development Client.
+`expo-dev-client` is an npm package installable in any Expo or React Native project. Once installed, any Debug builds of your application will gain an extensible debug menu and the ability to load projects from Expo CLI. Release builds of your application will not change other than the addition of a few header files.
 
-## Documentation
+Your debug builds can be shared with anyone on your team who needs to work on or review your application. Your team can develop the JavaScript portion of your application with expo-cli and your custom client without waiting for your native code to build until the
+next time you need to upgrade, install a new module, or otherwise change the native code in your project.
```

Comment:
```suggestion
Your debug builds can be shared with anyone on your team who needs to work on or review your application. Your team can develop the JavaScript portion of your application with expo-cli and your custom client without waiting for your native code to build until the next time you need to upgrade, install a new module, or otherwise change the native code in your project.
```

Reply:
(removed unnecessary newline)